content/posts/2015-02-02_february-newsletter.md

```diff
@@ -0,0 +1,130 @@
+Title: February Newsletter
+Date: 2015-02-02
+Tags: newsletter
+Slug: february-newsletter-2015
+Author: David Fischer
+
+
+Howdy,
+
+If there are articles, jobs or events you'd like to see in the next newsletter
+or if you have any feedback on San Diego Python, feel free to
+[share with us][].
+
+[share with us]: mailto:sandiegopython-organizers@googlegroups.com
+
+
+Upcoming events
+---------------
+
+[Saturday study group][saturday-meetup] (meetup.com) <br />
+San Diego PyLadies and SD Python are continuing the Saturday study group
+at Pangea cafe. The goal of the Saturday study group is to learn Python
+together in a collaborative environment. Over the next few weeks, the group
+will feature some data mining/analysis presentations by Eric Busboom. If
+you're going to the data science workshop, check it out!
+
+[saturday-meetup]: http://www.meetup.com/pythonsd/events/219919725/
+
+
+[Monthly Meetup][] (meetup.com) <br />
```

Comment:
Hi Carol,
Is the topic the same as your poster/demo for the Data Science Day on Feb
28th ?
Thanks
--Alain

On Mon, Feb 2, 2015 at 11:19 AM, Carol Willing <notifications@github.com>
wrote:

> In content/posts/2015-02-02_february-newsletter.md
> <https://github.com/pythonsd/pythonsd.org/pull/65#discussion_r23949831>:
>
> > +
> > +
> > +Upcoming events
> > +---------------
> > +
> > +[Saturday study group][saturday-meetup] (meetup.com) <br />
> > +San Diego PyLadies and SD Python are continuing the Saturday study group
> > +at Pangea cafe. The goal of the Saturday study group is to learn Python
> > +together in a collaborative environment. Over the next few weeks, the group
> > +will feature some data mining/analysis presentations by Eric Busboom. If
> > +you're going to the data science workshop, check it out!
> > +
> > +[saturday-meetup]: http://www.meetup.com/pythonsd/events/219919725/
> > +
> > +
> > +[Monthly Meetup][] (meetup.com) <br />
>
> On 02/02/2015 10:51, David Fischer wrote: In content/posts/
> 2015-02-02_february-newsletter.md <
> https://github.com/pythonsd/pythonsd.org/pull/65#discussion_r23947481>: >
> + > + > +Upcoming events > +--------------- > + > +[Saturday study
> group][saturday-meetup] (meetup.com) <br /> > +San Diego PyLadies and SD
> Python are continuing the Saturday study group > +at Pangea cafe. The goal
> of the Saturday study group is to learn Python > +together in a
> collaborative environment. Over the next few weeks, the group > +will
> feature some data mining/analysis presentations by Eric Busboom. If >
> +you're going to the data science workshop, check it out! > + >
> +[saturday-meetup]: http://www.meetup.com/pythonsd/events/219919725/ > +
> > + > +[Monthly Meetup][] (meetup.com) <br /> Were you thinking something
> on the order of 5-10 minutes? Or something longer? — Reply to this email
> directly or view it on GitHub <
> https://github.com/pythonsd/pythonsd.org/pull/65/files#r23947481>.
> Was hoping for about 25 minutes... Ideally a dry run for PyCon with enough
> time to fine tune.
> … <#14b4bbbe0e066865_>
> -- *Carol Willing* Developer | Willing Consulting
> https://willingconsulting.com
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pythonsd/pythonsd.org/pull/65/files#r23949831>.
>



-- 
--Alain
619-884-5450